addons/mrp/models/mrp_workorder.py

```diff
@@ -147,6 +147,7 @@ def _read_group_workcenter_id(self, workcenters, domain, order):
     json_popover = fields.Char('Popover Data JSON', compute='_compute_json_popover')
     show_json_popover = fields.Boolean('Show Popover?', compute='_compute_json_popover')
     consumption = fields.Selection(related='production_id.consumption')
+    qty_carried = fields.Float('Carried Quantity', digits='Product Unit of Measure', copy=False)
```

Comment:
`qty_carried` is the quantity already processed by previous workorders, isn't ?
I find the name a little bit vague. Maybe `qty_already_processed` or something like that could be more meaningful ?